### DISCUSSION TITLE
Most Visited Sites to Open in New Tab

### DISCUSSION THREAD

**Discussion by jeffchavez-dev** [dailydotdev/daily#1168] [Feature Requests]
Good day,
I added my most visited sites. Is it possible to enable opening the most visited in a new tab?
I haven't seen this feature.

Regards,
Jeff

**Comment by idoshamun** [comment]
Hi @jeffchavez-dev,

Currently it's not possible, but since it's a native link object you can middle click it to open in new tab.
I hope it's a useful workaround
